apps/condo/domains/organization/integrations/sbbol/sync/index.js

```diff
@@ -52,7 +52,7 @@ const { syncTokens } = require('./syncTokens')
 /**
  * Creates or updates user, organization, subscription, tokens from SBBOL callback data
  * @param keystone
- * @param {UserInfo} userInfo
+ * @param {UserInfo} userInfo data from OAuth client about user
```

Comment:
`{UserData}`

Reply:
Here is `userInfo`.
`userData` is in `syncOrganization`

Reply:
oke 👌